apps/publications/src/app/dialogs/add-authors-dialog/add-authors-dialog.component.html

```diff
@@ -0,0 +1,63 @@
+<div (keyup)="onKeyInput($event)">
+  <h1 class="page-subtitle">{{'DIALOGS.ADD_AUTHORS.TITLE' | translate}}</h1>
+
+  <div>
+    <mat-form-field class="mr-2 search-field no-border-mat-input">
+      <label>
+        <input matInput
+               autocomplete="false"
+               placeholder="{{'DIALOGS.ADD_AUTHORS.SEARCH_PLACEHOLDER' | translate}}"
+               [formControl]="searchControl">
+      </label>
+      <mat-error>
+        {{'DIALOGS.ADD_AUTHORS.EMPTY_SEARCH_MESSAGE' | translate }}
+      </mat-error>
+    </mat-form-field>
+    <button mat-flat-button
+            color="primary"
+            class="mr-2"
+            [disabled]="loading || searchControl.value.trim() === ''"
+            (click)="onSearchByString()">
+      <mat-icon>search</mat-icon>
+    </button>
+  </div>
+  <mat-spinner *ngIf="loading" class="ml-auto mr-auto">
+  </mat-spinner>
+  <div *ngIf="!loading">
+    <perun-web-apps-authors-add-list
+      *ngIf="authors !== undefined && authors.length > 0"
+      [authors]="authors"
+      [selection]="selection"
+      [selectedAuthors]="authorsToAdd"
+      [pageSize]="pageSizeAuthors"
+      [displayedColumns]="['select', 'id', 'name', 'organization']"
+      [disableRouting]="true"
+      (addAuthors)="authorsToAddChanged($event)"
+      (page)="pageChangedAuthors($event)">
+    </perun-web-apps-authors-add-list>
```

Comment:
With this component I really think that you made a lot of unneeded work.
Wouldnt it be easier if you put another two columns to authors-list(add button, cancel button).
on the "pick to add" author list you would show add column with add button which would emit output event to the add-authors-dialog.component where it would add the author to the addAuthors property which would just show another authors-list table - there would be simply cancel button with similar logic to remove author from adding.
The checkboxes are not really UX nice to do logic like that.